JASP-Desktop/components/JASP/Widgets/ModuleInstaller.qml

```diff
@@ -70,7 +70,7 @@ Popup
 					margins:	jaspTheme.generalAnchorMargin
 				}
```

Comment:
This file if full of strings without qsTr. I suppose you don't think this is necessary for this version?

Reply:
Well, no not really it is going to be replaced entirely anyway right?
By the module library?

Reply:
And anyway, I only changed it to change the functioncall, I didn't even look at the other lines of code.